diff --git a/src/records/schema.rs b/src/records/schema.rs
index 9a4cfc5..fd76a0a 100644
--- a/src/records/schema.rs
+++ b/src/records/schema.rs
@@ -11,11 +11,11 @@ diesel::table! {
 diesel::table! {
     records (id) {
         id -> Integer,
         task -> Text,
         project_id -> Integer,
-        started_at -> Timestamp,
-        ended_at -> Nullable<Timestamp>,
+        started_at -> TimestamptzSqlite,
+        ended_at -> Nullable<TimestamptzSqlite>,
     }
 }
 
 diesel::joinable!(records -> projects (project_id));
