diff --git a/src/schema.rs b/src/schema.rs
index 9a4cfc5..fd76a0a 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -10,10 +10,10 @@ diesel::table! {
 diesel::table! {
     records (id) {
         id -> Integer,
         task_id -> Integer,
-        started_at -> Timestamp,
-        ended_at -> Nullable<Timestamp>,
+        started_at -> TimestamptzSqlite,
+        ended_at -> Nullable<TimestamptzSqlite>,
     }
 }
 
 diesel::table! {
